vignettes/adrs.Rmd

```diff
@@ -314,6 +312,57 @@ adrs <- adrs %>%
   )
 ```
 
+```{r, echo=FALSE}
+dataset_vignette(
+  adrs,
+  display_vars = exprs(USUBJID, AVISIT, PARAMCD, AVALC, ADT, ANL01FL, ANL02FL)
+)
+```
+
+### Select Source Assessments for Parameter derivations
+
+For most parameter derivations the post-baseline overall response assessments up
+to and including first PD are considered.
+```{r}
+ovr <- filter(adrs, PARAMCD == "OVR" & ANL01FL == "Y" & ANL02FL == "Y")
+```
+
+```{r, echo=FALSE}
+dataset_vignette(
+  ovr,
+  display_vars = exprs(USUBJID, AVISIT, AVALC, ADT, RANDDT)
+)
+```
+
+## Define Events
+
+Common events like complete response (CR), partial response (PR), ... are
```

Comment:
```suggestion
The building blocks for the events that contribute to deriving common endpoints like what constitutes a responder, or a Best Overall Response of complete response (CR), ... are
```
I feel like this would read better

Reply:
Updated